index.js

```diff
@@ -84,8 +84,8 @@ module.exports = {
 	*/
 	fromSwarm: function (swarmHash) {
 		swarmHash = hexString(swarmHash)
-		let multihash = multiH.encode(swarmHash, 'keccak-256')	// get Multihash buffer
-		let res = new CID(1, 'dag-pb', multihash)				// create a CIDv1 with the multihash
+		let multihash = multiH.encode(swarmHash, 'bmt')	// get Multihash buffer
```

Comment:
@nonsense please change to `keccak256` in light of discussion on https://github.com/ensdomains/ens-app/issues/113

Reply:
@justelad I will as soon as a final conclusion on `bmt` vs `keccak256` is reached. :)

Reply:
@justelad , @pldespaigne fixed. I believe this is ready to be merged.